packages/playwright-core/src/utils/isomorphic/locatorGenerators.ts

```diff
@@ -194,7 +192,32 @@ function innerAsLocators(factory: LocatorFactory, parsed: ParsedSelector, isFram
     let locatorPartWithEngine: string | undefined;
     if (['xpath', 'css'].includes(part.name)) {
       const selectorPart = stringifySelector({ parts: [part] }, /* forceEngineName */ true);
-      locatorPartWithEngine = factory.generateLocator(base, locatorType, selectorPart);
+      locatorPartWithEngine = factory.generateLocator(base, 'default', selectorPart);
+    }
+
+    if (nextPart && nextPart.name === 'internal:control' && (nextPart.body as string) === 'enter-frame') {
+      // two options plus engine name:
+      // - locator('iframe').contentFrame()
+      // - locator('css|xpath=iframe').contentFrame()
+      // - frameLocator('iframe')
+      // - frameLocator('css|xpath=iframe')
+
+      const contentFrame = factory.generateLocator(base, 'frame', '')
+      const options = [
+        factory.chainLocators([locatorPart, contentFrame]),
+        factory.generateLocator(base, 'frame-locator', selectorPart),
+      ]
+
+      if (locatorPartWithEngine)
```

Comment:
style: missing curly braces around multiline `if` body.

Reply:
You're right! Seems like there is a bug with the `curly` rule in eslint, where it doesn't recognize multiline function calls as being multiline. The rule seems to be frozen though, so no benefit in reporting it